build.psm1

```diff
@@ -1081,26 +1081,6 @@ function Test-PSPesterResults
 function Start-PSxUnit {
     [CmdletBinding()]param()
 
-    log "xUnit tests are currently disabled pending fixes due to API and AssemblyLoadContext changes - @andschwa"
-    return
-
-    if ($Environment.IsWindows) {
-        throw "xUnit tests are only currently supported on Linux / OS X"
-    }
-
-    if ($Environment.IsMacOS) {
-        log "Not yet supported on OS X, pretending they passed..."
-        return
-    }
-
-    # Add .NET CLI tools to PATH
-    Find-Dotnet
```

Comment:
This should probably stay before any calls made to `dotnet`.

Reply:
resolved

Reply:
It looks like Find-Dotnet is still missing in Start-PSxUnit. It should probably be in any function that calls dotnet to ensure it is in the path. For example, Publish-PSTestTools